source/docs/05-mensural.xml

```diff
@@ -165,33 +165,33 @@
                   <p>The following example illustrates an <hi rend="italic">alteration</hi> (the second <hi rend="italic">brevis</hi>) in <hi rend="italic">modus minor perfectus</hi>. Notice that the second <hi rend="italic">brevis</hi> has doubled its regular value, it has been altered, unlike the first one. <figure>
-                        <head/>
+                        <head>Example on alteration: The bottom staff, together with the dotted barlines, is used here to help visualizing the durational values of the notes in the upper staff.</head>
                         <egXML xmlns="http://www.tei-c.org/ns/Examples" rend="verovio code" xml:space="preserve"><xi:include xmlns:xi="http://www.w3.org/2001/XInclude" href="../examples/verovio/alteration.mei" parse="text"/></egXML>
                      </figure>
                   </p>
                   <p>It is possible to omit the <att>dur.quality</att> attribute in a note when the performed duration of the note is the default value provided by the mensuration. In this case, the <hi rend="italic">longas</hi> are perfect, just as the mensuration (<hi rend="italic">perfect modus minor</hi>) indicates. Therefore, the <att>dur.quality</att> attribute can be omitted for the two <hi rend="italic">longas</hi>. <figure>
-                        <head/>
+                        <head>Example of omitting <att>dur.quality</att> for default values provided by the mensuration.</head>
                         <egXML xmlns="http://www.tei-c.org/ns/Examples" rend="code" xml:space="preserve"><xi:include xmlns:xi="http://www.w3.org/2001/XInclude" href="../examples/mensural/mensural-sample164.txt" parse="text"/></egXML>
                      </figure> The same applies to the examples that follow.</p>
                   <p>The following example illustrates an <hi rend="italic">imperfection</hi> (the two <hi rend="italic">longae</hi>) in <hi rend="italic">modus minor perfectus</hi> with the same <hi rend="italic">longa</hi>-<hi rend="italic">brevis</hi>-<hi rend="italic">brevis</hi>-<hi rend="italic">longa</hi> sequence but with an additional <hi rend="italic">dot of division</hi> between the two <hi rend="italic">breves</hi> (see <ptr target="#dots"/> for more details). Notice that here the <hi rend="italic">longae</hi> have been imperfected, unlike the previous example in which they kept the perfect value indicated by the mensuration. <figure>
-                        <head/>
+                        <head>Example on imperfection: The bottom staff, together with the dotted barlines, is used here to help visualizing the durational values of the notes in the upper staff.</head>
```

Comment:
```suggestion
                        <head>Example on imperfection (The bottom staff, together with the dotted barlines, is used here to help visualizing the durational values of the notes in the upper staff.)</head>
```

Reply:
I suggest:

"Example omitting <att>dur.quality</att> for default values provided by the mensuration." 
and 
"Imperfection: the bottom staff, together with the dotted barlines, is used to help visualize the durational values of the notes in the upper staff."